tephra-core/src/main/java/org/apache/tephra/distributed/thrift/TTransactionServer.java

```diff
@@ -68,6 +68,8 @@
 
     public TBoolean canCommitTx(TTransaction tx, Set<ByteBuffer> changes) throws TTransactionNotInProgressException, org.apache.thrift.TException;
 
+    public TBoolean canCommitOrThrow(TTransaction tx, Set<ByteBuffer> changes) throws TTransactionNotInProgressException, TGenericException, org.apache.thrift.TException;
```

Comment:
please do not review this file, it is generated by Thrift. Only need to review transaction.thrift